### PR TITLE
[plugin.video.worldstarhiphop] 1.0.14+matrix.1

### PR DESCRIPTION
**Pull request by skipmodea1**
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

**Comment by skipmodea1** [comment]
hello?

**Comment by skipmodea1** [comment]
@enen92 @basrieter Hija if you have time, could you look at some Pull Requests here? Thanks!

File: plugin.video.worldstarhiphop/resources/lib/worldstarhiphop_list.py [hunk 1]
```diff
@@ -94,21 +94,16 @@ def getVideos(self):
         # Parse response
```
Comment (enen92) [inline plugin.video.worldstarhiphop/resources/lib/worldstarhiphop_list.py:94]: please drop the pyo compiled file

**Review by enen92 — CHANGES_REQUESTED** [review]
Code looks good, binary files have to go

File: plugin.video.worldstarhiphop/resources/lib/worldstarhiphop_list.py (hunk 1 above)
Comment (skipmodea1) [reply plugin.video.worldstarhiphop/resources/lib/worldstarhiphop_list.py:94]: When i look at the files, the pyo files are already not there?!

Comment (enen92) [reply plugin.video.worldstarhiphop/resources/lib/worldstarhiphop_list.py:94]: nevermind, no idea how I missed them on the last submission. We should add this to the addon-checker

**Comment by skipmodea1** [comment]
Ah my bad, i don't know how to remove the pyo files from this PR. I'll make a new PR.

**Review by enen92 — APPROVED** [review]

**Comment by enen92** [comment]
No need to, in this PR you are actually removing the files.

**Pull request closed without merging**